src/Extend/ApiSerializer.php

```diff
@@ -67,13 +67,35 @@ public function attribute(string $name, $callback)
      *
      * @return self
      */
-    public function mutate($callback)
+    public function attributes($callback)
```

Comment:
I just realized, we need to deprecate the `mutate` method, since that was released in the previous version

Reply:
it is deprecated, look below this code

Reply:
I.... don't know how I missed that... What.... :see_no_evil: